src/main/java/com/boun/swe/mnemosyne/model/Memory.java

```diff
@@ -0,0 +1,88 @@
+package com.boun.swe.mnemosyne.model;
+
+
+import lombok.Builder;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.JoinTable;
+import javax.persistence.ManyToMany;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+import javax.persistence.Temporal;
+import javax.persistence.TemporalType;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import java.util.Date;
+import java.util.Set;
+
+@Getter
+@Setter
+@EqualsAndHashCode
+@Builder
+@Entity
+@Table(name = "memories")
+public class Memory {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(name = "id")
+    private Long id;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "user_id", nullable = false)
+    private User user;
+
+    @Size(max = 300)
+    @NotNull
+    private String title;
+
+    @Size(max = 12000)
```

Comment:
Bro, is this a restriction of hibernate?

Reply:
No, I don't think there is any restriction. I'll be removing this maximum argument definition.